networkstatsquerier/fetcher/fetcher.go

```diff
@@ -6,28 +6,36 @@ import (
 	"os"
 )
 
-// Keyword is a struct for each keyword
+// OrgKeyword is a struct for each keyword
 type OrgKeyword struct {
-	Kind  string `json:"Kind"`
-	Value string `json:"Value"`
+	Kind  string
+	Value string
 }
 
 // EndpointEntry is a struct for each entry of data pulled out of the EndpointSources file
 type EndpointEntry struct {
-	OrganizationName     string       `json:"OrganizationName"`
-	FHIRPatientFacingURI string       `json:"FHIRPatientFacingURI"`
-	Type                 string       `json:"Type"`
-	Keywords             []OrgKeyword `json:"Keywords"`
+	OrganizationName     string
+	FHIRPatientFacingURI string
+	ListSource           string
+	Type                 string
+	Keywords             []OrgKeyword
```

Comment:
thoughts on possibly removing these? because they are not wide-spread, i don't see us using them. I can go either way, though.

Reply:
i'm fine with removing them!

Reply:
👍